### PR TITLE
Fix count on string

### LINKED ISSUE #12614: [PHP 7.2] count(): Parameter must be an array or an object that implements Countable
### Steps to reproduce
1. Upgrade to Nextcloud 14.04
2. Access NC

### Expected behaviour
Normal access to nextcloud without errors in error log

### Actual behaviour
Access to nextcloud works, but there are errors in the log:
count(): Parameter must be an array or an object that implements Countable at /var/www/nextcloud/apps/user_ldap/lib/Access.php#883

### Server configuration

**Operating system**: Ubuntu 18.04

**Web server:** Nginx

**Database:** MySQL

**PHP version:** 7.2

**Nextcloud version:** 14.04

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated from 14.03

### PR DESCRIPTION
**Pull request by kesselb**
Fix #12614

### CODE REVIEW

**Review by rullzer — DISMISSED** [review]

File: apps/user_ldap/lib/Access.php [hunk 1]
```diff
@@ -2011,4 +2011,17 @@ private function initPagedSearch($filter, $bases, $attr, $limit, $offset) {
 		return $pagedSearchOK;
 	}
 
+	/**
+	 * Is more than one $attr used for search?
+	 *
+	 * @param string|string[]|null $attr
+	 * @return bool
+	 */
+	private function manyAttributes($attr) {
```
[the remark this answers is not in the document]
Comment (danielkesselberg) [reply apps/user_ldap/lib/Access.php]: image

:see_no_evil: :hear_no_evil:

**Review by blizzz — APPROVED** [review]
makes sense, thx!

**Comment by blizzz** [comment]
@rullzer mind a re-review?

**Review by MorrisJobke — APPROVED** [review]
Code makes sense 👍

**Comment by MorrisJobke** [comment]
> @rullzer 😕

Should be fixed by https://github.com/rullzer/backportbot/pull/12

**Pull request merged**